### DISCUSSION TITLE
3 Party Apps

### DISCUSSION THREAD

**Discussion by ooTruffle** [moudey/Shell#116] [General]
When shell registers itself it appears on some 3 party apps

On Openshell it adds options that dont need to be there
![image](https://user-images.githubusercontent.com/28374251/188297701-687cb535-6aa9-45fc-9c86-1648cb96b3d2.png)
is there any way to modify this menu / remove certain enteries

**Comment by moudey** [comment]
Most Explorer context menus are handled by Shell with editing and others by redrawing only.  But a new option will be added to allow editing other menus that appear through the Explorer or a third party loaded by it.
